paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -1247,46 +1293,40 @@ def get_splunk_api_token(self) -> SplunkAuth:
         )
 
     def get_button_text(self, button: str, is_active: bool) -> str:
-        # Button text max length 75 characters
-        # Current button templates allow version max length of 36
-        version_short_str = self.deployment_version.short_sha_repr()
-        if len(version_short_str) > 36:
-            # we'll have to depend on subsequent slack messages to show full version
-            version_short_str = "new version"
+        # TODO: Update button text for image_version
+        #  Will require thought on shortening
         active_button_texts = {
-            "forward": f"Rolling Forward to {version_short_str} :zombocom:"
+            "forward": f"Rolling Forward to {self.commit[:8]} :zombocom:"
         }
         inactive_button_texts = {
-            "forward": f"Continue Forward to {version_short_str} :arrow_forward:",
-            "complete": f"Complete deploy to {version_short_str} :white_check_mark:",
+            "forward": f"Continue Forward to {self.commit[:8]} :arrow_forward:",
+            "complete": f"Complete deploy to {self.commit[:8]} :white_check_mark:",
             "snooze": f"Reset countdown",
             "enable_auto_rollbacks": "Enable auto rollbacks :eyes:",
             "disable_auto_rollbacks": "Disable auto rollbacks :close_eyes_monkey:",
         }
 
-        if self.old_deployment_version is not None:
-            old_version_short_str = self.old_deployment_version.short_sha_repr()
-            # Current button templates allow old version max length 43
-            if len(old_version_short_str) > 43:
-                old_version_short_str = "old version"
+        # TODO: Update to check image_version also
```

Comment:
should we make a ticket for this?

Reply:
Not needed anymore as this was needed in the previous version where we used `commit[:8]` and not `version_short_str`.